code/game/machinery/telecomms/machines/message_server.dm

```diff
@@ -127,11 +124,6 @@
  * it in this proc, you have to do so yourself.
  */
 /obj/machinery/telecomms/message_server/proc/generate_key()
```

Comment:
do subtypes use this proc somehow? i don't get why you stubbed it out while moving all the code it contains elsewhere